base/db/R/insert.format.vars.R

```diff
@@ -48,18 +48,20 @@ insert.format.vars <- function(con, format_name, mimetype_id, notes = NULL, head
   }
 
   # Test if format name already exists
-  name_test <- dplyr::tbl(con, "formats") %>% dplyr::select("id", "name") %>% dplyr::filter(.data$name %in% !!format_name) %>% dplyr::collect()
-  name_test_df <- as.data.frame(name_test)
-  if(!is.null(name_test_df[1,1])){
+  name_test <- dplyr::tbl(con, "formats") %>%
+    dplyr::select(.data$id, .data$name) %>%
```

Comment:
Use of .data inside select() is [deprecated](https://github.com/r-lib/tidyselect/issues/169) -- this trips me up too; the distinction is between "data-masking contexts" and "tidy selection contexts". But long story short, use quoted strings inside select now.
```suggestion
    dplyr::select("id", "name") %>%
```